src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -0,0 +1,53 @@
+package dataloader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.TreeMap;
+
+/**
+ * Class that takes a String as path file
+ * to csv files
+ * and creates datasets in two formats
+ * TreeMap & Table from tablesaw.
+ */
+
+public class CsvDataLoader {
+
+    /**
+     * Simple method that parses data set from a csv file
+     * without knowing the headers of the file
+     */
+    public void parseFile(String csvFile, TreeMap dataSet) throws IOException {
+
+        Reader csvData = new FileReader(csvFile);
+
+        CSVParser parser = CSVParser.parse(csvData, CSVFormat.DEFAULT);
+
+        for (CSVRecord record : parser) {
+            for (String field : record) {
+                dataSet.keySet();
+                dataSet.get(field);
+
+                System.out.println("\"" + field + "\", ");
+            }
+        }
+    }
+
+    /**
+     * Reads from csv in file system with columns separated by commas
+     * and the file has a header row.
+     * Missing values are treated with an indicator.
+     */
+    public void parseFile(String csvFile) throws IOException {
+        //dataSet.read().usingOptions(CsvReadOptions.builder(csvFile).missingValueIndicator("-"));
```

Comment:
Look at this:

```
CsvReadOptionsBuilder builder = 
	CsvReadOptions.builder("myFile.csv")
		.separator('\t')										// table is tab-delimited
		.header(false)											// no header
		.dateFormat("yyyy.MM.dd");  				// the date format to use. 

CsvReadOptions options = builder.build();

Table t1 = Table.read().usingOptions(options);
```

Reply:
Does sth like this works for us? You call the ```builder``` but you don't seem to build the options....

Reply:
I think it will work now. Check out the latest commit. Also the tests are passing. Both of them as far as I can understand.